test/basics.jl

```diff
@@ -748,4 +748,15 @@ f = CSV.File(IOBuffer("a,b\n1,2\n3,"))
 @test f.a == [1, 3]
 @test isequal(f.b, [2, missing])
 
+# 951
+data = """
+| Name       |  Zip |
+| Joe        |  123 |
+| Mary Anne  | 1234 |
+"""
+f = CSV.File(IOBuffer(data); delim='|', normalizenames=true, stripwhitespace=false)
+@test f.Name[1] == " Joe        "
+f = CSV.File(IOBuffer(data); delim='|', normalizenames=true, stripwhitespace=true)
```

Comment:
You can remove `normalizenames` in this case, right? Since `stripwhitespace` will result in column names like `Name` and `Zip`, right? It might be good to clarify this in the keyword doc; i.e. that the whitespace stripping also applies to parsing the column names.

Reply:
Good point. I'll update the test and documentation.

Reply:
```suggestion
f = CSV.File(IOBuffer(data); delim='|', stripwhitespace=true)
```